index.js

```diff
@@ -65,18 +65,45 @@ menuLinkContactElement.addEventListener('click', () => {
 
 // ----------------------- contact validation ---------------------------------
 const formElement = document.getElementById('contact-form');
-// const formNameElement = document.getElementById('name');
+const formNameElement = document.getElementById('name');
 const formEmailElement = document.getElementById('email');
 
 formElement.addEventListener('submit', (event) => {
   const email = formEmailElement.value;
   const errmessages = [];
   if (email.toLowerCase() !== email) {
     event.preventDefault();
-    errmessages.push('email must all be in lowercase letters. form was not submitted');
+    errmessages.push(
+      'email must all be in lowercase letters. form was not submitted',
+    );
     const emailErrElement = document.getElementById('email-err');
-    // Destructure the entire `errmessages` array and access the first element using an index
     const [firstErrorMessage] = errmessages;
     emailErrElement.innerText = firstErrorMessage;
   }
 });
+
+const formData = {
+  commentElement: document.getElementById('comment-box'),
```

Comment:
- [ ] The message is never stored in the local storage, kindly make sure to fix this so the message is stored in the local storage